UPDATING.md

```diff
@@ -23,6 +23,11 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Removal of StatsD metrics `local_task_job_heartbeat_failure`
+
+New metrics named `lower(JobClassName)_heartbeat_failure` will replacing the removed metric.
```

Comment:
It's now `localtaskjob_heartbeat_failure`, right? Could you add that example?

Or we could keep the pattern the same to make this a seamless upgrade:

```
re.sub('(?!^)([A-Z]+)', r'_\1', self.__class__.__name__).lower()
```

Reply:
That sub looks good, will do that.